app/portainer/rbac/components/access-viewer/access-viewer.html

```diff
@@ -25,8 +25,8 @@
 
         <div class="col-sm-12 form-section-title"> Access </div>
         <div>
-          <div class="small text-muted" style="margin-bottom: 15px">
-            <i class="fa fa-info-circle blue-icon space-right" aria-hidden="true"></i>
+          <div class="small text-muted vertical-center" style="margin-bottom: 15px">
+            <pr-icon icon="'alert-circle'" mode="'primary'" feather="true" size="'sm'" class="vertical-center"></pr-icon>
```

Comment:
icon here is wrong, should use `info` from feather according to our icon guide 
![image](https://user-images.githubusercontent.com/12197793/182048959-610467e7-5503-4bb0-8978-99fc42118d91.png)